TitanEngine/TitanEngine.Debugger.DebugLoop.cpp

```diff
@@ -1237,7 +1237,7 @@ __declspec(dllexport) void TITCALL DebugLoop()
             //general unhandled exception callback
             if(DBGCode == DBG_EXCEPTION_NOT_HANDLED)
             {
-                engineStepActive = false;
+                engineStepTID = 0;

```

Comment:
Maybe we should only clear the field if the unhandled exception comes from the same thread that's awaiting the step to complete?